src/platforms/esp32/components/avm_builtins/spi_driver.c

```diff
@@ -181,15 +181,15 @@ Context *spi_driver_create_port(GlobalContext *global, term opts)
     term hspi_atom = globalcontext_make_atom(global, ATOM_STR("\x4", "hspi"));
 
     term bus_config = interop_kv_get_value(opts, ATOM_STR("\xA", "bus_config"), global);
-    term miso_io_num_term = interop_kv_get_value(bus_config, ATOM_STR("\xB", "miso_io_num"), global);
-    term mosi_io_num_term = interop_kv_get_value(bus_config, ATOM_STR("\xB", "mosi_io_num"), global);
+    term miso_term = interop_kv_get_value(bus_config, ATOM_STR("\x4", "miso"), global);
+    term mosi_term = interop_kv_get_value(bus_config, ATOM_STR("\x4", "mosi"), global);
     term sclk_io_num_term = interop_kv_get_value(bus_config, ATOM_STR("\xB", "sclk_io_num"), global);
```

Comment:
Is `sclk` changing here, as well?  Should this be changed to `sclk_term` (and the atom to `sclk`)